cuqi/sampler/_conjugate_approx.py

```diff
@@ -0,0 +1,52 @@
+from cuqi.distribution import Posterior, Laplace_diff, Gamma
+import numpy as np
+import scipy as sp
+
+class ConjugateApprox:
```

Comment:
To be subclassed from sampler?

Reply:
Yes, once we get an updated and improved Sampler class.